docs/msf/managing-store-configuration.mdx

```diff
@@ -41,36 +47,51 @@ Accept: application/json
 }
 ```
 
+</Tab>
+</Tabs>
+
 Because the request did not specify any query parameters, the API returned the **global default** configuration for this merchant. Responses that return global configuration data will contain values for all properties of the requested object.
 
 The merchant's global configuration is used for all storefronts and other channels that have not specified different, channel-specific configuration settings.
 
 To check whether channel-specific values for a setting are defined, send the `channel_id` as a query parameter in your requests to settings endpoints.
 
-```http title="Example request: Get store profile settings for channel 122"
+<Tabs items={['Request', 'Response']}>
+<Tab>
+
+```http filename="Example request: Get store profile settings for channel 122"
 GET https://api.bigcommerce.com/stores/{{STORE_HASH}}/v3/settings/profile?channel_id=122
 X-Auth-Token: {{ACCESS_TOKEN}}
 Accept: application/json
 ```
-&nbsp;
-```json title="Example response: Get store profile settings for channel 122" lineNumbers
+</Tab>
+<Tab>
+
+```json filename="Example response: Get store profile settings for channel 122" showLineNumbers
```

Comment:
```suggestion
```json filename="Example response: Get store profile settings for channel 122" showLineNumbers copy
```